### PR TITLE
migrate star 2.0.2 to Vidarr

### PR DESCRIPTION
**Pull request by xuemeiluo**

### CODE REVIEW

File: star.wdl [hunk 1]
```diff
@@ -1,47 +1,27 @@
...
-   description: "STAR 2.1"
-   dependencies: [
+meta {
+ author: "Peter Ruzanov"
+ email: "peter.ruzanov@oicr.on.ca"
+ description: "STAR 2.0"
+ dependencies: [
       {
-        name: "star/2.7.6a",
+        name: "star/2.7.3a",
```
Comment (pruzanov) [inline star.wdl:24]: I am confused. The CHANGELOG says: update to 2.7.6a, but the version in the commit is 2.7.3a? README also says 2.7.6a, something needs to be adjusted

File: star.wdl [hunk 2]
```diff
@@ -75,26 +55,21 @@ input {
   String chimericjunctionSuffix = "Chimeric.out"
   String genereadSuffix = "ReadsPerGene.out"
   String? addParam
-  String modules = "star/2.7.6a hg38-star-index100/2.7.6a"
-  String chimOutType = "WithinBAM HardClip Junctions"
-  Int outFilterMultimapNmax = 50
-  Int chimScoreDropMax = 30
+  String modules = "star/2.7.3a hg38-star-index100/2.7.3a"
```
Comment (pruzanov) [inline star.wdl:58]: This also looks suspicious. If we are going to use 2.7.3a and not 2.7.6a the README and CHANGELOG need to be changed

**Review by pruzanov — CHANGES_REQUESTED** [review]
Need to make clear which version is used (in README and CHANGELOG) or change the modules/versions in wdl

File: star.wdl (hunk 1 above)
Comment (xuemeiluo) [reply star.wdl:24]: This is the item that I added in the changelog
## 2.0.4 - 2022-06-22
- migrate star 2.0.2 to Vidarr

File: star.wdl (hunk 2 above)
Comment (xuemeiluo) [reply star.wdl:58]: I didn't update README, but I can do it. This PR is to migrate an old star workflow version 2.0.2 (not the latest one) to Vidarr.

Comment (xuemeiluo) [reply star.wdl:58]: updated

**Review by mlaszloffy — APPROVED** [review]

**Review by alexjfortuna — APPROVED** [review]

**Review by pruzanov — APPROVED** [review]

**Pull request closed without merging**